backend/migration.js

```diff
@@ -0,0 +1,229 @@
+const path = require('path');
+const Umzug = require('umzug');
+const _ = require('lodash');
+
+function onMigrationEnd(exitCode) {
+    // eslint-disable-next-line no-process-exit
+    process.exit(exitCode);
+}
+
+function getCurrent(executed) {
+    return _.chain(executed).last().get('file').value() || '<NO_MIGRATIONS>';
+}
+
+/**
+ * Runs migration script
+ *
+ * @param {Object} loggerFactory initialized logger factory
+ * @param {Object} dbModule DB module, an object containing `init` function and `db` object
+ */
+function runMigration(loggerFactory, dbModule) {
+    const { db, init } = dbModule;
+
+    const command = process.argv[2].trim();
```

Comment:
I know it's part of already existing code, but:
- maybe it would be wiser to push process args to the function parameters?
- if not, then maybe worth to secure it from the error when there is no `argv[2]`? (similarly L185 could be secured)

Reply:
Done